testdata/fuzz/FuzzWrite/caf81e9797b19c76

```diff
@@ -0,0 +1,2 @@
+go test fuzz v1
```

Comment:
what does this file do?

Reply:
it is a file hat is generated from the fuzz tests. It is used for regression testing as input data for the FizzWrite test.